app/services/is_weekend_or_holiday.rb

```diff
@@ -0,0 +1,5 @@
+class IsWeekendOrHoliday
+  def self.call(date)
+    IsWeekend.call(date) || HolidayService.observed_holiday?(date)
+  end
+end
```

Comment:
This seems like a great method to just add on `HolidayService`

```
HolidayService.is_weekend?
HolidayService.is_weekend_or_holiday?
```

Reply:
Also no test coverage for this class?

Reply:
HolidayService is a rebel.  It was before we standardized on one method per service class.

Reply:
I think it is disadvantageous to have one method for service class, it's so much easier to read and find when there are multiple related methods on the same class

Reply:
yeah, I think consolidating in the holiday service is easier vs. having to navigate 2 or 3 files

separately, we could consider inverting non-weekend and non-holiday to be business/work day or something?

Reply:
I guess it can continue being a rebel but it definitely doesn't make sense to add `is_weekend_or_holiday` to `HolidayService`

Reply:
would it be ok to rename to `CalendarService`?

Reply:
Now you're talking :-)